backends-velox/src/test/scala/io/glutenproject/execution/VeloxLiteralSuite.scala

```diff
@@ -62,4 +71,70 @@ class VeloxLiteralSuite extends VeloxWholeStageTransformerSuite {
     validateOffloadResult("SELECT struct(X'1234')")
     validateOffloadResult("SELECT struct(DATE'2020-12-31')")
   }
+
+  test("Array Literal") {
+    validateOffloadResult("SELECT array('Spark', '5')")
+    validateOffloadResult("SELECT array(5, 1, -1)")
+    validateOffloadResult("SELECT array(5S, 1S, -1S)")
+    validateOffloadResult("SELECT array(5Y, 1Y, -1Y)")
+    validateOffloadResult("SELECT array(true, false)")
+    validateOffloadResult("SELECT array(1D, -1D)")
+    validateOffloadResult("SELECT array(1F, -1F)")
+    validateOffloadResult("SELECT array(1.0, 5.321)")
+    validateOffloadResult("SELECT array(5.321E2BD, 5.321E2BD)")
+    validateOffloadResult("SELECT array(5.321E2BD, 0.1, 5.321E22BD)")
+    validateOffloadResult("SELECT array(TIMESTAMP'2020-12-31', TIMESTAMP'2020-12-30')")
+    validateOffloadResult("SELECT array(X'1234', X'a')")
+    validateOffloadResult("SELECT array(DATE'2020-12-31', DATE'2020-12-30')")
+    validateOffloadResult("SELECT array(array(3, 4), array(5, 6))")
+    validateOffloadResult("SELECT array(map(1,2,3,4))")
+    validateOffloadResult("SELECT array(map('red', 1), map('green', 2))")
+    validateOffloadResult("SELECT array(struct(6, 'test1'), struct(5, 'test'))")
+  }
+
+  test("Map Literal") {
+    validateOffloadResult("SELECT map('b', 'a', 'e', 'e')")
+    validateOffloadResult("SELECT map(1D, 'a', 2D, 'e')")
+    validateOffloadResult("SELECT map(1.0, map(1, 2, 3, 4))")
```

Comment:
Do we need to add map(map, map) ?

Reply:
Using `map` as key of a map is not allowed in spark. Thanks.